### PR TITLE
Release notes 2022.2

### PR DESCRIPTION
**Pull request by pmli**

### CODE REVIEW

File: docs/source/release_notes/2022.2.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+by the reductor such that |LTIModels| of different orders can be constructed
+efficiently.
+
+Parametric AAA algorithm
+~~~~~~~~~~~~~~~~~~~~~~~~
+The AAA algorithm allows for approximating rational functions in an iterative
+manner by combining the ideas of Vector-Fitting and Loewner interpolation. With
+this release we are adding the parametric version of the algorithm
+(`[#1756] p-AAA <https://github.com/pymor/pymor/pull/1756>`_) for data-driven
+approximation of |TransferFunctions|. The |PAAAReductor| can handle any models
```
Comment (artpelling) [inline docs/source/release_notes/2022.2.rst:40]:
```suggestion
approximation of |TransferFunctions|. The :class:`PAAARedcutor <pymor.reductors.aaa.PAAAReductor` can handle any models
```

File: docs/source/release_notes/2022.2.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+of the Markov parameters for cases where there are many inputs and/or outputs.
+The SVD of the Hankel matrix as well as the tangential projectors are cached
+by the reductor such that |LTIModels| of different orders can be constructed
+efficiently.
+
+Parametric AAA algorithm
+~~~~~~~~~~~~~~~~~~~~~~~~
+The AAA algorithm allows for approximating rational functions in an iterative
+manner by combining the ideas of Vector-Fitting and Loewner interpolation. With
+this release we are adding the parametric version of the algorithm
```
Comment (artpelling) [inline docs/source/release_notes/2022.2.rst:38]:
```suggestion
this release, we are adding the parametric version of the algorithm
```

File: docs/source/release_notes/2022.2.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+with different right-hand sides.
+
+DWR reductor for output estimation with primal-dual approach
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#1496] DWR reductor for output estimation with primal-dual approach <https://github.com/pymor/pymor/pull/1496>`_
+
+|LTIModel| can compute LQG and BR Gramians
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The :meth:`~pymor.models.iosys.LTIModel.gramian` of the |LTIModel| so far could
+only compute the standard, Lyapunov Gramians.
```
Comment (artpelling) [inline docs/source/release_notes/2022.2.rst:87]:
```suggestion
only compute the standard Lyapunov Gramians.
```

**Review by artpelling — CHANGES_REQUESTED** [review]

**Comment by sdrave** [comment]
@TiKeil, can you add a few lines regarding the DWR stuff.

**Review by sdrave — APPROVED** [review]
Fine from my side.

Depending on whether or not the impulse response PR makes it into the release or not, the respective part of the release notes has to be adapted. Also, the release date still needs to be specified.

**Review by artpelling — APPROVED** [review]

**Pull request merged**